conjure-core/src/test/java/com/palantir/conjure/defs/validator/FieldNameValidatorTest.java

```diff
@@ -55,18 +55,22 @@ public void testInvalidNames() throws Exception {
         }) {
             assertThatThrownBy(() -> FieldNameValidator.validate(FieldName.of(invalid)))
                     .isInstanceOf(IllegalArgumentException.class)
-                    .hasMessageContaining(
-                            String.format("FieldName \"%s\" must follow one of the following patterns", invalid));
+                    .hasMessageContaining(String.format(
+                            "FieldName \"%s\" must follow one of the following patterns: "
+                                    + "[LOWER_CAMEL_CASE[^[a-z]([A-Z]{1,2}[a-z0-9]|[a-z0-9])*[A-Z]?$], "
+                                    + "KEBAB_CASE[^[a-z]((-[a-z]){1,2}[a-z0-9]|[a-z0-9])*(-[a-z])?$], "
+                                    + "SNAKE_CASE[^[a-z]((_[a-z]){1,2}[a-z0-9]|[a-z0-9])*(_[a-z])?$]]",
+                            invalid));
```

Comment:
Why did we change this? It may create unnecessary work if we later update these patterns. We don't particularly care to test this part of the message exactly. Although I'm generally in favor of validating full messages.

Did you intend to change `hasMessageContaining` to `hasMessage`?

Reply:
Ah good point.  I'd wanted to confirm I hadn't changed / broken this part of the message, but pasting the full contents makes it a [change detector test](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html).  

Instead I've now rewritten the assertions to assert it contains `LOWER_CAMEL_CASE` and `[A-Z]` as a measure of containing the name of the regexes plus the contents of the regexes